### PR TITLE
Update tagesschau.de.xml

### PR DESCRIPTION
**Pull request by heuser443**
Add more domains

### CODE REVIEW

File: src/chrome/content/rules/tagesschau.de.xml [hunk 1]
```diff
@@ -46,7 +56,11 @@
 		to="https://www.tagesschau.de/"/>
 
 		<test url="http://www.tagesschau.de/" />
-
+	
+	<!-- "301 Moved Permanently", safe to redirect -->
```
Comment (J0WI) [inline src/chrome/content/rules/tagesschau.de.xml]: Please add a comment why this special rule is required.

File: src/chrome/content/rules/tagesschau.de.xml [hunk 2]
```diff
@@ -46,7 +56,11 @@
 		to="https://www.tagesschau.de/"/>
```
Comment (J0WI) [inline src/chrome/content/rules/tagesschau.de.xml]: Please add a comment why this special rule is required.

File: src/chrome/content/rules/tagesschau.de.xml [hunk 3]
```diff
@@ -27,13 +30,20 @@
...
+	<target host="download.media.tagesschau.de"/>
+		<test url="http://download.media.tagesschau.de/video/2017/1127/TV-20171127-2021-1301.h264.mp4" />
+	<target host=          "meta.tagesschau.de"/>
+	<target host=       "service.tagesschau.de"/>
+		<test url="http://service.tagesschau.de/novi/" />
+	<target host=          "wahl.tagesschau.de"/>
+	<target host=        "wetter.tagesschau.de"/>
+	<target host=             "tagesschau24.de"/>
+	<target host=         "www.tagesschau24.de"/>
+	<target host=    "programm.tagesschau24.de"/>
```
Comment (J0WI) [inline src/chrome/content/rules/tagesschau.de.xml]: This code style is odd. Could you remove the spaces within the xml tags?

**Comment by DavidLiedke** [comment]
@J0WI this PR can be closed. 

PR was replaced by #14681

**Pull request closed without merging**